docs/source/ui/pages/run_management.md

```diff
@@ -138,9 +138,11 @@ Use the bottom controllers to control the steps and the indices of the tracked t
 
 ### Figures
 
-Aim allows to track Plotly and matplotlib figures.
+Aim allows tracking Plotly and matplotlib figures.
 On the `Figures` tab you can view all the track figures over different contexts and steps.
 
+**Note:** Aim will render figures with passed or default dimensions and due to that container can be scrollable.
```

Comment:
Aim will render figures with passed or default dimensions. There will be scrolls if the size exceeds the plotly container space of the Figures tab. 🤔